### PR TITLE
TBOX-111 add renameFields parameter

### PR DESCRIPTION
**Pull request by shalinipriya0965**
# ↪️ Pull Request

## ✔️ PR Todo

- [ ] Add documentation
- [ ] Add examples
- [ ] Ensure you are following the practices in the [contributor guide](https://docs.google.com/document/d/1c_hhpDMhO0zN793Z_HFOz42Q4g7l1nPRVQ5pXOUsMJU/edit) and [coding_standards](https://datatamr.atlassian.net/wiki/spaces/FEKB/blog/2020/04/14/1137606830/Customer+Success+Coding+Standards)
  * [Use consistent and descriptive name conventions](https://docs.google.com/document/d/1c_hhpDMhO0zN793Z_HFOz42Q4g7l1nPRVQ5pXOUsMJU/edit#heading=h.477s36w4rds9) 
  * [Uses google-style docstrings](https://datatamr.atlassian.net/wiki/spaces/FEKB/blog/2020/04/14/1137606830/Customer+Success+Coding+Standards#Google-style-docstrings)
  * [Uses versioned API/client methods only](https://docs.google.com/document/d/1c_hhpDMhO0zN793Z_HFOz42Q4g7l1nPRVQ5pXOUsMJU/edit#heading=h.yv6df99fyrq2)
  * [Avoid global variables](https://datatamr.atlassian.net/wiki/spaces/FEKB/blog/2020/04/14/1137606830/Customer+Success+Coding+Standards#Avoid-global-variables)
  * [Avoid giant try/catch](https://datatamr.atlassian.net/wiki/spaces/FEKB/blog/2020/04/14/1137606830/Customer+Success+Coding+Standards#Avoid-giant-try-/-except)
  * [Don't use mutable default variables](https://datatamr.atlassian.net/wiki/spaces/FEKB/blog/2020/04/14/1137606830/Customer+Success+Coding+Standards#Don%E2%80%99t-use-mutable-default-arguments)
  * [Use a `main()` function in your scripts](https://datatamr.atlassian.net/wiki/spaces/FEKB/blog/2020/04/14/1137606830/Customer+Success+Coding+Standards#Use-a-%E2%80%9Cmain()%E2%80%9D--function-in-your-scripts)
- [ ] Added/updated testing for this change (ensure you are testing any changes/new code!)
- [ ] Update title to link to TBOX issue (i.e. start pull request title with TBOX-xx) 
- [x] Approval from first code-reviewer
- [ ] Approval from second code-reviewer (only required if a second reviewer has left comments)
- [ ] Pass all checks

### CODE REVIEW

File: tests/data_io/test_connect.py [hunk 1]
```diff
@@ -91,6 +91,7 @@ def test_get_export_config_default():
         "mergedArrayValuesDelimiter": "|",
         "limitRecords": 0,
         "columnsExcludeRegex": "",
+        "renameFields": None,
```
Comment (wpmartin) [inline tests/data_io/test_connect.py]: Can you change the default to an empty Dict ("{}") to match the connect repo examples: https://github.com/Datatamer/core-connect#client-credentials-example-1

File: tests/data_io/test_connect.py [hunk 2]
```diff
@@ -100,6 +101,7 @@ def test_get_export_config_with_args():
         "mergedArrayValuesDelimiter": ",",
         "limitRecords": 10,
         "columnsExcludeRegex": "(origin).*",
+        "renameFields": None,
```
Comment (wpmartin) [inline tests/data_io/test_connect.py]: here too

File: tests/data_io/test_connect.py (hunk 1 above)
Comment (shalinipriya0965) [reply tests/data_io/test_connect.py]: Done

File: tests/data_io/test_connect.py (hunk 2 above)
Comment (shalinipriya0965) [reply tests/data_io/test_connect.py]: Done

**Review by wpmartin — APPROVED** [review]
lgtm

**Pull request merged**